gearcatalog/frontend/src/actions/actions.types.ts

```diff
@@ -1,7 +1,9 @@
 import { Gear, GearNoId } from '../components/gear/types';
 
 // TODO: #9 Create correct type for User
-export type User = Record<string, unknown>;
+export type User = {
+  username: string,
```

Comment:
Users don't have IDs?

Reply:
They do, I just don't use them on the frontend as of right now. I figured I would just type what I'm currently using.